README.md

```diff
@@ -157,32 +157,28 @@ Though it would never be displayed as part of an aggregated feed unless requeste
 This is how it might be stored in a database system:
 
     "accessControls":	{
-    	"access":   {
-    		"keys":		[
-    			"secretkey123",
-    			"secretkey456"	
-    		],
-    		"users":	[
-    			"root",
-    			"john"
-    		],
-    		"roles":	[
-    			"employee"
-    		]
-    	},
-    	"admin": {
-    		"keys":		[
-    			"supersecretkey12345"
-    		],
-    		"users":	[
-    			"root"
-    		],
-    		"roles":    [
-    			"manager"
-    		]
-    	}
+        "users": {
+            "john": [
+            	"read"
+            ],
+            "fred": [
+                { "read": "deny" }
+            ]
+        },
+        "roles": {
+            "staff": [
+                "read"
+            ],
+            "admin": [
+                "write"
+            ]
+        }
```

Comment:
I think read/write should always be objects so that they are predictable.  I'd prefer something like:

```
john: [
  { read: true }
],
fred: [
  { read: false }
]
```

or could replace true/false with "allow"/"deny", but I like true/false better cuz booleans are easier to deal with.